scripts/test-runner.sh

```diff
@@ -48,4 +50,4 @@ else
 fi
 
 
-yarn "$BACKEND:start:background" && yarn $TEST_COMMAND && yarn "$BACKEND:stop"
+yarn "$BACKEND:start:background" && yarn generate:accounts && yarn $TEST_COMMAND && yarn "$BACKEND:stop"
```

Comment:
Just wondering why do we need to `generate:accounts` at the start of the backend?
Thanks,

Reply:
We refill accounts from one main account in tests. So when we run in parallel, we've received the nonce error. That's why we need to add a list of accounts with refilled balances before running tests. And then we can use it one by one in parallel tests